node_modules/oae-core/userdetails/css/userdetails.css

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+@media (min-width: 769px) and (max-width: 920px) {
+    #userdetails-modal .modal-dialog {
+        margin: 0 10px;
+        width: auto;
+    }
+}
+
+@media (min-width: 920px) {
+    #userdetails-modal .modal-dialog.termsandconditions {
+        width: 900px;
+    }
+}
+
+/*
+ * By default, show the user details form
+ */
+#userdetails-modal #userdetails-form {
+    display: block;
+}
+#userdetails-modal #userdetails-terms-and-conditions-container {
+    display: none;
+}
+#userdetails-modal .modal-footer {
+    display: none;
+}
+
+/*
+ * If the `termsandconditions` class is added to the modal, we show
+ * the Terms and Conditions view
+ */
+#userdetails-modal.termsandconditions #userdetails-form {
+    display: none;
+}
+#userdetails-modal.termsandconditions #userdetails-terms-and-conditions-container {
+    display: block;
+}
+#userdetails-modal.termsandconditions .modal-footer {
+    display: block;
+}
+
+#userdetails-modal .well {
+    margin-bottom: 0;
+    max-height: 300px;
+    overflow: auto;
+}
+
+#userdetails-save {
+    margin-top: 10px;
+}
+
+#userdetails-form-accept-terms-and-conditions-container {
+    margin-top: 10px;
```

Comment:
I'm surprised that this is necessary. Appropriate use of `form-group` should make this unnecessary